### ISSUE TITLE
Hover and pressed states on navigation bar

### ISSUE THREAD

**Issue by Poopooracoocoo** [ADeltaX/MobileShell#16]
The hover state would be neat on laptops with a touchscreen.

**Comment by ShadowEO** [comment]
I agree with this, would be neat for tablets with pens too (because they can provide the hover state)! I was looking into implementing it in my personalized fork but couldn't figure it out myself (I'm only a XAML novice). I think this combined with accent personalization (that's real easy to get from Windows too, it's a single line to get the color brush for Win32 based XAML applications) would almost complete the facsimile of the Windows desktop shell.
